pom.xml

```diff
@@ -92,6 +92,10 @@
         <enabled>false</enabled>
       </snapshots>
     </repository>
+    <repository>
```

Comment:
AFAIK, it looks like it's always been needed and it's been only because the local build agents already had snapshots in their Maven cache that it worked.